docs/source-fabric/fundamentals/accelerators.rst

```diff
@@ -3,6 +3,13 @@ Accelerate your code with Fabric
 ################################
 
 
+.. raw:: html
+
+    <center>
+        <video width="100%" max-width="500px" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/accelerators.mp4", type="video/mp4" autoplay loop muted></video>
```

Comment:
https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/accelerators.mp4

Reply:
pro tip: if you install https://github.com/refined-github/refined-github then links become clickable in the diff

Reply:
Nice. I guess I still did a good job copying the links for all those who don't have the extension 😄